### ISSUE TITLE
Unexpected HTTP response: 404 when trying to download packwiz

### ISSUE THREAD

**Issue by aninusmuffin** [ParadigmMC/mc-modpack-kit#76]
**Describe the bug**
The github action is trying to build the pack but https://nightly.link/packwiz/packwiz/workflows/go/main/Linux%2064-bit%20x86.zip gives a 404 response

```
Run supplypike/setup-bin@v3
  with:
    uri: https://nightly.link/packwiz/packwiz/workflows/go/main/Linux%2064-bit%20x86.zip
    name: packwiz
    version: linux
  env:
    pack-file: pack.toml
    default-release-modrinth: true
    default-release-curseforge: false
    loader: quilt
    game-version: 1.20.2
Error: Error: Unexpected HTTP response: 404
```

**Additional context**

- https://github.com/Aninuscsalas/wynncraft-101/actions/runs/8456718187/job/23167185436#step:9:257

**Comment by sofiedotcafe** [comment]
Hey!
This appears to be an upstream issue, see here https://github.com/packwiz/packwiz/actions/runs/7317810289
Even on the newest release, they are expired?

I can probably just use the nix pkg they have provided, it complies it from source

**Comment by sofiedotcafe** [comment]
Should be fixed with #77
